autobuild/windows/autobuild_windowsinstaller_2_build.ps1

```diff
@@ -27,4 +28,11 @@ if (("$jamulus_project_path" -eq $null) -or ("$jamulus_project_path" -eq "")) {
 
 echo "Build installer..."
 # Build the installer
-powershell "$jamulus_project_path\windows\deploy_windows.ps1" "C:\Qt\5.15.2"
+if ($BuildOption -Eq "jackonwindows")
```

Comment:
`-Ne """` - that way, whatever option is passed in gets passed through.

Reply:
Sounds good....to me. Will update the PR.